src/app/login.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Layout from './layout';
+
+var UserLogin = React.createClass({
```

Comment:
You should use ES6-style classes here. See [this doc](https://facebook.github.io/react/docs/reusable-components.html#es6-classes)